### PR TITLE
#16 Tie in frontend and new camera API

### LINKED ISSUE #16: Tie in frontend and new camera API

### PR DESCRIPTION
**Pull request by meowmeowcute**
*load cameras list from /camera/list
*camera id's type is now number instead of string

closes #16

### CODE REVIEW

File: frontend/src/app/cameras/camera/camera.service.ts [hunk 1]
```diff
@@ -5,12 +5,13 @@ import {HttpClient} from '@angular/common/http';
 import {map} from 'rxjs/operators';
 
 import CAMERAS from './mock-cameras';
-const CAMERA_URL = '';
+const CAMERA_URL = 'http://10.101.1.23:9090/camera/list';
```
Comment (viktorzdanovich) [inline frontend/src/app/cameras/camera/camera.service.ts]: would be great if camera url considers ip and port from `app/environment/environment.ts`

File: frontend/src/app/cameras/camera/camera.service.ts [hunk 2]
```diff
@@ -5,12 +5,13 @@ import {HttpClient} from '@angular/common/http';
...
 
 @Injectable({
   providedIn: 'root'
 })
 export class CameraService {
+  private cameras: any[] = [];
```
Comment (viktorzdanovich) [inline frontend/src/app/cameras/camera/camera.service.ts]: `private cameras: Camera[]` looks better 🙂

File: frontend/src/app/cameras/camera/camera.service.ts (hunk 1 above)
Comment (meowmeowcute) [reply frontend/src/app/cameras/camera/camera.service.ts]: fixed :)

File: frontend/src/app/cameras/camera/camera.service.ts (hunk 2 above)
Comment (meowmeowcute) [reply frontend/src/app/cameras/camera/camera.service.ts]: fixed

**Pull request merged**